docs/src/app/components/pages/components/FlatButton/ExampleComplex.jsx

```diff
@@ -30,13 +30,14 @@ const FlatButtonExampleComplex = () => (
       icon={<ActionAndroid />}
     />
 
-    <a href="https://github.com/callemall/material-ui">
-      <FlatButton
-        label="GitHub Link"
-        secondary={true}
-        icon={<FontIcon className="muidocs-icon-custom-github" />}
-      />
-    </a>
+    <FlatButton
```

Comment:
the `a` is breaking the layout:
![capture d ecran 2016-01-19 a 11 37 10](https://cloud.githubusercontent.com/assets/3165635/12416314/fe08aa8a-bea0-11e5-9dd7-e7c4492bca26.png)
What do you think about adding a `<br />` here?